spec/hydra/pcdm/ancestor_checker_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+RSpec.describe Hydra::PCDM::AncestorChecker do
+  subject { described_class.new(record) }
+
+  describe "#ancestor?" do
+    let(:record) { instance_double(Hydra::PCDM::Object) }
+    let(:member) { record }
+    let(:result) { subject.ancestor?(member) }
+
+    context "when the member is the record itself" do
+      it "should be true" do
```

Comment:
Consider avoiding "shoulds" in test descriptions: http://betterspecs.org/#should